front/public/locales/en/errors.json

```
@@ -49,7 +49,8 @@
     "infra": {
       "NotFound": "",
       "edition": {
-        "InfraIsLocked": "Infra is locked"
+        "InfraIsLocked": "Infra is locked",
+        "SplitTrackSectionBadOffset": "Distance to split trackSection '{{tracksection_id}}' in infra '{{infra_id}}' is invalid. It must be between 0 and {{tracksection_length}} meters"
```

We should always use the full word even in english (a issue will be done for harmonization in the rest of the app)
```suggestion
        "InfraIsLocked": "Infrastructure is locked",
        "SplitTrackSectionBadOffset": "Distance to split trackSection '{{tracksection_id}}' in infrastructure '{{infra_id}}' is invalid. It must be between 0 and {{tracksection_length}} meters".
```